### PR TITLE
chr pos split js safer for STR view

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Some issues remained / arose with filter lock with the new release candidate.

1. A js error on the STR filters (no chr_pos to split) disabled both populating chr and filter lock

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/variants/static/form_scripts.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
 function populateCytobands(cytobands){
   var chromPosPattern = new RegExp("^(?:chr)?([1-9]|1[0-9]|2[0-2]|X|Y|MT)$");
   var chrom = document.forms["filters_form"].elements["chrom"].value;
-  var chromPos = document.forms["filters_form"].elements["chrom_pos"].value;
+  var chromPos = "";
+  if (typeof document.forms["filters_form"].elements["chrom_pos"] != "undefined") {
```
Comment (dnil) [inline scout/server/blueprints/variants/static/form_scripts.js:6]: It's not just Jinja and Python! Hm, lets add rust next perhaps?

Comment (northwestwitch) [inline scout/server/blueprints/variants/static/form_scripts.js:6]:
```suggestion
  if (typeof document.forms["filters_form"].elements["chrom_pos"] !== "undefined") {
```

Comment (northwestwitch) [reply scout/server/blueprints/variants/static/form_scripts.js:6]: Nice that you found the problem! You probably tried this solution and it works, so feel free to ignore my suggestion. But I was thinking that if you are doing a type comparison perhaps the triple equals could be more appropriate than then double equal way (check the docs [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) for a description of the differences)

**Review by northwestwitch — APPROVED** [review]
Nice solution! 👍🏻

File: scout/server/blueprints/variants/static/form_scripts.js (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/variants/static/form_scripts.js:6]: Thanks, yes, that appears correct, though there is no real issue with using the double equal here either - typeof will return a string regardless of what came in.

**Pull request merged**